editor/editor_resource_preview.cpp

```diff
@@ -35,6 +35,7 @@
 #include "core/io/resource_loader.h"
 #include "core/io/resource_saver.h"
 #include "core/object/message_queue.h"
+#include "core/variant/variant_utility.cpp"
```

Comment:
Including the `.cpp` here causes a conflict with #70080 which also included the `.cpp` for the tests. I opened #78108 to make a header for this file so we can stop including the `.cpp` file.